Automattic/wp-calypso
# Checkout: use a minimal masterbar

Keep the user focused on the checkout task rather than the extra possiblities afforded by the Masterbar during checkout.

<img width="1050" alt="image" src="https://user-images.githubusercontent.com/195089/43483342-042ceba2-94d1-11e8-84a3-aa74da35fe3a.png">


## To Test

1. Add a plan to your cart
2. Go to checkout, the Masterbar should now only contain the "My Site(s)" `<Item>`
3. After completing a purchase (`/checkout/thank-you/…` ) the masterbar should be restored

## Other Notes

The checkout screen is the only place that the Masterbar should go minimal, so try a variety of other navigation scenarios to ensure it doesn't happen there.

There were previous tests for this in #2635 and #2558
